mllib/src/main/scala/org/apache/spark/mllib/evaluation/MultilabelMetrics.scala

```diff
@@ -142,17 +124,17 @@ class MultilabelMetrics @Since("1.2.0") (predictionAndLabels: RDD[(Array[Double]
     if((p + r) == 0) 0.0 else 2 * p * r / (p + r)
   }
 
-  private lazy val sumTp = tpPerClass.foldLeft(0L) { case (sum, (_, tp)) => sum + tp }
-  private lazy val sumFpClass = fpPerClass.foldLeft(0L) { case (sum, (_, fp)) => sum + fp }
-  private lazy val sumFnClass = fnPerClass.foldLeft(0L) { case (sum, (_, fn)) => sum + fn }
+  private lazy val sumTp = summary.tpPerClass.foldLeft(0L) { case (sum, (_, tp)) => sum + tp }
```

Comment:
I wonder why these use `foldLeft`? isn't it just a sum over values in the map, like `summary.tpPerClass.values.sum`?